Add exclusion filters and uninstall to App Sec docs (#13124)
DataDog/documentation: content/en/security_platform/application_security/setup_and_configure.md

The Datadog Agent and some tracing libraries have options available to address these situations and modify or discard spans. See [APM Data Security][1] for details that also apply to Application Security.

> ## Exclusion filters

Author: @arnaudbreton Bom dia! 👋  Opened this PR and added exclusion filters / uninstall content. Exclusion filters are not billed, correct? And is there a limit on the number of filters you can set? I'll add those notes in the PR once confirmed. Feel free to add any suggestions to this area, and uninstallation as well.

Reviewer: Indeed, requests (traces) matching an exclusion filter won't be charged. 

About the limit of filters, I don't think so but I'm asking to the team. [Slack thread](https://dd.slack.com/archives/C01UQV7HA3A/p1645012496120029)

There may be a time when an Application Security signal is a false positive. For example, a targeted service is repeatedly generating the same signal, but the signal has been reviewed and is not a threat.

You can set an exclusion filter, which ignore events from a rule, to eliminate these noisy signal patterns and focus on legitimate suspicious requests.

To create an exclusion filter, do one of the following:

- Click on a signal in [Application Security Signals][2] and click the **Create Exclusion Filter** button in the top left corner. This method automatically generates a filter query for the targeted service.
- Navigate to [Exclusion Filters Configuration][3] and manually configure a new exclusion filter based on your own filter query.

## Uninstall

> To uninstall Application Security, remove the `DD_APPSEC_ENABLED=true` environment variable from your application configuration. If you need additional help, contact [Datadog support].

Author: ```suggestion
To uninstall Application Security, remove the `DD_APPSEC_ENABLED=true` environment variable from your application configuration. If you need additional help, contact [Datadog support][4].
```

## Further Reading

{{< partial name="whats-next/whats-next.html" >}}

[1]: /tracing/setup_overview/configure_data_security/
[2]: https://app.datadoghq.com/security/appsec/signals

> [3]: https://app.datadoghq.com/security/appsec/exclusions

Author: ```suggestion
[3]: https://app.datadoghq.com/security/appsec/exclusions
[4]: /help/
```

The Datadog Agent and some tracing libraries have options available to address these situations and modify or discard spans. See [APM Data Security][1] for details that also apply to Application Security.

## Exclusion filters

> There may be a time when an Application Security signal is a false positive. For example, a targeted service is repeatedly generating the same signal, but the signal has been reviewed and is not a threat.

Author: ```suggestion
There may be a time when an Application Security signal is a false positive. For example, Application Security repeatedly detects
the same suspicious request and a signal is generated, but the signal has been reviewed and is not a threat.
```

> You can set an exclusion filter, which ignore events from a rule, to eliminate these noisy signal patterns and focus on legitimate suspicious requests.

Reviewer: > There may be a time when an Application Security signal is a false positive. For example, a targeted service is repeatedly generating the same signal, but the signal has been reviewed and is not a threat.

A few language tweaks to consider:
- A service doesn't generate signals. AppSec detects signals based on traffic it receives

> You can set an exclusion filter, which ignore events from a rule, to eliminate these noisy signal patterns and focus on legitimate suspicious requests.
Ignore security events from the event rules

To create an exclusion filter, do one of the following:

- Click on a signal in [Application Security Signals][2] and click the **Create Exclusion Filter** button in the top left corner. This method automatically generates a filter query for the targeted service.
- Navigate to [Exclusion Filters Configuration][3] and manually configure a new exclusion filter based on your own filter query.

> ## Uninstall

Reviewer: Disable?

> ## Uninstall

Author: ```suggestion
## Disable
```

> To uninstall Application Security, remove the `DD_APPSEC_ENABLED=true` environment variable from your application configuration. If you need additional help, contact [Datadog support][4].

Reviewer: The service must be restarted for the change to be taken in account

> To uninstall Application Security, remove the `DD_APPSEC_ENABLED=true` environment variable from your application configuration. If you need additional help, contact [Datadog support][4].

Author: ```suggestion
To disable Application Security, remove the `DD_APPSEC_ENABLED=true` environment variable from your application configuration. Once removed, restart your service.

If you need additional help, contact [Datadog support][4].
```

The Datadog Agent and some tracing libraries have options available to address these situations and modify or discard spans. See [APM Data Security][1] for details that also apply to Application Security.

## Exclusion filters

There may be a time when an Application Security signal, or a suspicious request, is a false positive. For example, Application Security repeatedly detects
the same suspicious request and a signal is generated, but the signal has been reviewed and is not a threat.

You can set an exclusion filter, which ignore events from a rule, to eliminate these noisy signal patterns and focus on legitimate suspicious requests.

To create an exclusion filter, do one of the following:

- Click on a signal in [Application Security Signals][2] and click the **Create Exclusion Filter** button in the top left corner. This method automatically generates a filter query for the targeted service.
- Navigate to [Exclusion Filters Configuration][3] and manually configure a new exclusion filter based on your own filter query.

**Note**: Requests (traces) matching an exclusion filter are not billed.

> ## Disable

Author: ```suggestion
## Disable Application Security
```
